lib/imagemagick/identify/result.rb

```diff
@@ -3,6 +3,7 @@
 
 module ImageMagick
   class Identify::Result
+    attr_reader :result
```

Comment:
This doesn't make a lot of sense, mainly because the class itself is a bit of a dumb class. The class is meant to represent the result, and have accessors to each attribute that we want (e.g. colors). The idea is to add each attribute as we go along so that the data is contained and converted/presented in a usable format.  Giving access to the results hash is not containing the data, so I'm going to close this PR.

If you submit another pull request with accessors to the exact attributes that you want to access within the result hash I'd be more than happy to merge that. :smile: